var/spack/repos/builtin/packages/py-tables/package.py

```diff
@@ -29,8 +29,8 @@ class PyTables(PythonPackage):
     variant('lzo', default=False, description='Support for lzo compression')
 
     # requirements.txt
-    depends_on('python@3.5:', when='@3.4:', type=('build', 'run'))
-    depends_on('python@2.6:', type=('build', 'run'))
+    depends_on('python@2.7:2.7.999,3.5:', when='@3.4:', type=('build', 'run'))
+    depends_on('python@2.6:2.6.999',  when='@:2',   type=('build', 'run'))
```

Comment:
These versions aren't in Spack

Reply:
I was trying to preserve the original version ranges as much as possible. Cleaned up.